x-pack/plugins/cases/public/containers/api.test.tsx

```diff
@@ -457,29 +456,32 @@ describe('Cases API', () => {
     });
   });
 
-  describe('getCaseUserActions', () => {
+  describe('findCaseUserActions', () => {
     beforeEach(() => {
       fetchMock.mockClear();
-      fetchMock.mockResolvedValue(caseUserActionsSnake);
+      fetchMock.mockResolvedValue(findCaseUserActionsSnake);
     });
 
     it('should be called with correct check url, method, signal', async () => {
-      await getCaseUserActions(basicCase.id, abortCtrl.signal);
-      expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/${basicCase.id}/user_actions`, {
+      await findCaseUserActions(basicCase.id, abortCtrl.signal);
+      expect(fetchMock).toHaveBeenCalledWith(`${CASES_URL}/${basicCase.id}/user_actions/_find`, {
         method: 'GET',
         signal: abortCtrl.signal,
+        query: {
+          perPage: MAX_DOCS_PER_PAGE,
+        },
       });
     });
 
     it('should return correct response', async () => {
-      const resp = await getCaseUserActions(basicCase.id, abortCtrl.signal);
-      expect(resp).toEqual(caseUserActions);
+      const resp = await findCaseUserActions(basicCase.id, abortCtrl.signal);
+      expect(resp).toEqual(findCaseUserActionsResponse);
     });
 
     it('should not covert to camel case registered attachments', async () => {
-      fetchMock.mockResolvedValue(caseUserActionsWithRegisteredAttachmentsSnake);
-      const resp = await getCaseUserActions(basicCase.id, abortCtrl.signal);
-      expect(resp).toEqual(caseUserActionsWithRegisteredAttachments);
+      fetchMock.mockResolvedValue(findCaseUserActionsSnake);
```

Comment:
I don't believe `findCaseUserActionsSnake` has any registered attachments in it, should we leave this as `caseUserActionsWithRegisteredAttachmentsSnake` and just switch the api call on 483?